apiserver/facades/controller/remoterelations/remoterelations_test.go

```diff
@@ -238,7 +238,7 @@ func (s *remoteRelationsSuite) TestGetTokens(c *gc.C) {
 }
 
 func (s *remoteRelationsSuite) TestSaveMacaroons(c *gc.C) {
-	mac, err := macaroon.New(nil, "id", "")
+	mac, err := macaroon.New(nil, []byte("id"), "")
```

Comment:
man, we need a helper for this i reckon. 
New which just takes 2 strings as the last 2 arguments, or even a single string and set location to ""
could add it along with the MacaroonEquals stuff